librarymanagement/src/main/scala/sbt/librarymanagement/RichUpdateReport.scala

```diff
@@ -17,13 +15,17 @@ final class RichUpdateReport(report: UpdateReport) {
   import DependencyFilter._
 
   /** Obtains all successfully retrieved files in all configurations and modules. */
-  def allFiles: Seq[File] = matching(DependencyFilter.allPass)
+  private[sbt] def allFiles: Seq[File] = matching(DependencyFilter.allPass)
 
   /** Obtains all successfully retrieved files in configurations, modules, and artifacts matching the specified filter. */
-  def matching(f: DependencyFilter): Seq[File] = select0(f).distinct
+  private[sbt] def matching(f: DependencyFilter): Seq[File] = select0(f).distinct
 
   /** Obtains all successfully retrieved files matching all provided filters.  An unspecified argument matches all files. */
-  def select(configuration: ConfigurationFilter = configurationFilter(), module: ModuleFilter = moduleFilter(), artifact: ArtifactFilter = artifactFilter()): Seq[File] =
+  def select(
+    configuration: ConfigurationFilter = configurationFilter(),
```

Comment:
+1 version with all 3 undefaulted too. That would cover the 3 use-cases I see in Scala.js.